src/clickup/__tests__/e2e.clickup.test.ts

```diff
@@ -0,0 +1,111 @@
+import { IMerjoonProjects, IMerjoonTasks, IMerjoonUsers } from '../../common/types';
+import { ClickUpService } from '../service';
+import { getClickUpService } from '../clickup-service';
+import { ID_REGEX } from '../../utils/regex';
+
+describe('e2e ClickUp', () => {
```

Comment:
add test similar to https://github.com/Merjoon/merjoon-core/pull/74/files#diff-0bfa85afe8b3e6c4fc93d2e6ed4b3bfd5f8a123a15851c07fb8e2c542ffea0a7R101